manifest-production.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+inherit: manifest-base.yml
+domain: registers-trial.service.gov.uk
```

Comment:
Is this section specific to the `registers-frontend` application? The inherited `manifest-base.yml` contains 3 applications so does this definitely override the correct one?